### PR TITLE
Allow calls to Apply3D(AudioListener[] listeners, AudioEmitter emitte…

### PR DESCRIPTION
**Pull request by bartwe**
Allow calls to Apply3D(AudioListener[] listeners, AudioEmitter emitter) with no or 1 AudioListener

### CODE REVIEW

**Comment by flibitijibibo** [comment]
Just needs style fixes and this is good to go:

Tabs, not spaces
Braces on all if/else blocks
Braces on the left

File: src/Audio/SoundEffectInstance.cs [hunk 1]
```diff
@@ -263,6 +263,12 @@ public void Apply3D(AudioListener listener, AudioEmitter emitter)
 
 		public void Apply3D(AudioListener[] listeners, AudioEmitter emitter)
 		{
+            if (listeners.Length == 0)
```
Comment (flibitijibibo) [inline src/Audio/SoundEffectInstance.cs]: I just took a second look and this actually isn't valid - if listeners is empty you get ArgumentOutOfRange. [1] is valid, however, so you can just take this first check out entirely.

**Comment by bartwe** [comment]
Hope this is better :)

**Comment by flibitijibibo** [comment]
Merged, thanks!

**Pull request merged**